hack/jenkins/job-configs/kubernetes-e2e-trusty.yaml

```diff
@@ -0,0 +1,75 @@
+- job-template:
+    name: 'kubernetes-e2e-gce-trusty-{suffix}'
+    description: '{description} Test owner: {test-owner}.'
+    logrotate:
+        daysToKeep: 7
+    builders:
+        - shell: |
+            curl -fsS --retry 3  "https://raw.githubusercontent.com/kubernetes/kubernetes/{branch}/hack/jenkins/e2e.sh" | bash -
+    properties:
+        - mail-watcher
+    publishers:
+        - claim-build
+        - junit-publisher
+        - log-parser
+        - email-ext:
+            recipients: "{emails}"
+        - description-setter:
+            regexp: KUBE_GCE_MINION_IMAGE=(.*)
+        - groovy-postbuild: |
+            def trustyImageMatcher = manager.getLogMatcher("KUBE_GCE_MINION_IMAGE=(.*)")
+            if(trustyImageMatcher?.matches()) manager.addShortText("<b>GCI Image: " + trustyImageMatcher.group(1) + "</b>", "grey", "white", "0px", "white")
+            def k8sVersionMatcher = manager.getLogMatcher("Using\\spublished\\sversion\\s(.*)\\s\\(from.*")
+            if(k8sVersionMatcher?.matches()) manager.addShortText("<br><b>Kubernetes version: " + k8sVersionMatcher.group(1) + "</b>", "grey", "white", "0px", "white")
+
+    triggers:
+        - timed: 'H H/8 * * *'
+    wrappers:
+        - ansicolor:
+            colormap: xterm
+        - timeout:
+            timeout: '{timeout}'
+            fail: true
+        - timestamps
+        - workspace-cleanup
+
+- project:
+    name: kubernetes-e2e-gce-trusty
+    test-owner: 'wonderfly@google.com'
+    branch: 'release-1.1'
+    emails:
```

Comment:
This should be a single comma-separated list. Ugly, I know, but it just puts it straight into the Jenkins config which expects it.

`emails: adityakali@google.com,ameyd@google.com,...`